ui/src/components/AppDetailedToolbar.css

```diff
@@ -0,0 +1,8 @@
+.details-toolbar{
+    margin-left: 3% 
+}
+.toolbar-buttons{
```

Comment:
Styling feedback:

- as already mentioned, some spacing between buttons needs to be added. 
- colour of buttons
  - should both be same colour
  - are chosen colours best to integrate with page

Reply:
I've used the default styling which comes with patternfly, i.e. the primary button. We also use the blue buttons in mdc so wanted to keep that consistent. I'm aware that @davidffrench is using a red button in one of the modals so I think we do have some creative licence here.